shims/common/src/main/scala/org/apache/gluten/config/GlutenConfig.scala

```diff
@@ -1716,4 +1717,11 @@ object GlutenConfig {
       .booleanConf
       .createWithDefault(true)
 
+  val COLUMNAR_CUDF_ENABLED =
+    buildConf("spark.gluten.sql.columnar.cudf")
+      .internal()
+      .doc("Enable or disable cudf support.")
+      .booleanConf
+      .createWithDefault(true)
```

Comment:
Several questions:

1. Could we add an "experimental" mark in the option `doc()`? 
2. Should we turn it off by default?
3. What happens from user side if CPP code is not built with `--enable_gpu` but this option is turned on?